sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 5.6.0 (Unreleased)
+
+**New Features**
+
+- Added support for sending AMQP annotated message which allows full access to the AMQP message fields.
+  - Introduced new namespace `azure.eventhub.amqp`.
+  - `azure.eventhub.amqp.AMQPAnnotatedMessage` is now made public and could be instantiated for sending.
```

Comment:
I don't think we need this line for EH? (IIRC there's no AMQPAnnotatedMessage in EH before)